pkg/server/chaosd/network.go

```diff
@@ -66,7 +66,7 @@ func (networkAttack) Attack(options core.AttackConfig, env Environment) (err err
 	case core.NetworkPortOccupiedAction:
 		return env.Chaos.applyPortOccupied(attack)
 
-	case core.NetworkDelayAction, core.NetworkLossAction, core.NetworkCorruptAction, core.NetworkDuplicateAction, core.NetworkBandwidthAction:
+	case core.NetworkDelayAction, core.NetworkLossAction, core.NetworkCorruptAction, core.NetworkDuplicateAction, core.NetworkBandwidthAction, core.NetworkPartitionAction:
```

Comment:
the `partition` is  deleted by mistake in https://github.com/chaos-mesh/chaosd/pull/91 😢